src/components/SignupPopup/SignupPopup.jsx

```diff
@@ -59,18 +105,20 @@ const SignupPopup = ({
           onChange={handleEmailChange}
```

Comment:
If you have `onChange` or ` onInput` in an input then you need to add attribute `value` to this input.
This is how React controls inputs.

To read more about **controlled components**: https://reactjs.org/docs/forms.html#controlled-components

**Please, correct it everywhere**